### PR TITLE
WIP: Initial fix for generating ROTM nomination activity for Mentor

### PR DESCRIPTION
**Pull request by CuriousLearner**
This is untested currently.

I wasn't able to access the admin. Facing some weird issue after login which sometimes says `Something went wrong. Please contact the administrator` and sometimes show CSRF failed.

Logs didn't show anything which would help me in debugging. Still figuring out with my local instance on this and would update here.

About the patch, it would be good to discuss the generic idea of placing things or if team prefers some other way on getting it done. 

Thanks :)

### CODE REVIEW

File: remo/profiles/forms.py [hunk 1]
```diff
@@ -309,4 +310,7 @@ def save(self, nominated_by, *args, **kwargs):
                 self.cleaned_data['is_rotm_nominee']):
             self.instance.is_rotm_nominee = True
             self.instance.rotm_nominated_by = nominated_by
+            generate_nomination_activity_for_mentor.apply_async(
+                kwargs={'mentor': nominated_by}
```
[the remark this answers is not in the document]
Comment (CuriousLearner) [reply remo/profiles/forms.py]: Agreed.

**Comment by Mte90** [comment]
My only question is about the nomination that has to be private.
In the code I am not sure of that point.

**Comment by MichaelKohler** [comment]
> My only question is about the nomination that has to be private.
In the code I am not sure of that point.

The "mentor" is passed as only argument. This is the user that has initiated the nomination, not the user that was nominated. Therefore the report will never include the nominated RoM.

**Comment by CuriousLearner** [comment]
@MichaelKohler We need to manually generate this activity once: `ACTIVITY_ROTM_NOMINATED`. I'm not sure how these activities are generated. I can create a custom migration for it if you want.

**Comment by MichaelKohler** [comment]
> @MichaelKohler We need to manually generate this activity once: ACTIVITY_ROTM_NOMINATED. I'm not sure how these activities are generated. I can create a custom migration for it if you want.

Sounds reasonable.

**Comment by CuriousLearner** [comment]
Sure, I'll put this in tomorrow for sure :)

**Comment by MichaelKohler** [comment]
> Sure, I'll put this in tomorrow for sure :)

quick ping here ;)

**Comment by CuriousLearner** [comment]
@MichaelKohler I've added a custom migration for now. Will address reviews later today. Sorry for the delay.

**Pull request closed without merging**